Process.c

```diff
@@ -1057,7 +1057,7 @@ void Process_updateExe(Process* this, const char* exe) {
    if (exe) {
       this->procExe = xStrdup(exe);
       const char* lastSlash = strrchr(exe, '/');
-      this->procExeBasenameOffset = (lastSlash && *(lastSlash + 1) != '\0' && lastSlash != exe) ? (size_t)(lastSlash - exe + 1) : 0;
+      this->procExeBasenameOffset = (lastSlash > exe && *(lastSlash + 1) != '\0') ? (size_t)(lastSlash - exe + 1) : 0;
```

Comment:
Side note: I have no idea why this conditional needs a `lastSlash != exe`. I thought it should be legitimate when `lastSlash == exe`, that is, when the `exe` string begins with a slash such as "`/vmlinuz`".